### ISSUE TITLE
compute node PATH is corrupted

### ISSUE THREAD

**Issue by bsmith94** [awslabs/scale-out-computing-on-aws#4]
I have launched a compute node in SOCA and logged in remotely. The path contains a space and a quote. This is a corrupt path setting.

My ami is based on centos 7.6. 

```
$ python3 ./add_nodes.py --instance_type m5.large --desired_capacity 1 --queue alwayson --keep_forever --instance_ami ami-0aed6074101673b41 --job_name always_on_capacity --job_owner bsmith
$ ssh 110.0.111.220
Warning: Permanently added '110.0.111.220' (ECDSA) to the list of known hosts.
Last login: Thu Feb 13 00:25:22 2020 from ip-110-0-27-85.ec2.internal

Compute Node Ready for queue: alwayson
$ echo $PATH
/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/pbs/bin:/opt/pbs/sbin:/opt/pbs/bin:/apps/python/latest/bin" :/opt/pbs/bin:/data/home/bsmith/.local/bin:/data/home/bsmith/bin
```

**Comment by bsmith94** [comment]
I forgot to mention that I had executed ```cp /etc/skel/.bash* ~``` to get the standard bash configuration when I login. It looks like that space in PATH is tripping up /etc/profile.d/pbs.sh.

**Comment by mcrozes** [comment]
Issue is fixed on 1.1. I merge your PR for 1.0. Thanks
